components/ui/calendar.tsx

```
@@ -0,0 +1,75 @@
+        caption_label: "text-sm font-medium",
+        nav: "space-x-1 flex items-center",
+        nav_button: cn(
+          buttonVariants({ variant: "outline" }),
+          "h-7 w-7 bg-transparent p-0 opacity-50 hover:opacity-100",
+        ),
+        button_previous:
+          "absolute hover:bg-hoverbg left-2 top-2 rounded-full border flex items-center justify-center p-1",
+        button_next:
+          "absolute hover:bg-hoverbg right-2 top-2 rounded-full border flex items-center justify-center p-1",
+        table: "w-full border-collapse space-y-1",
+        weekdays: "flex",
```

Biome (1.9.4)

[error] This property value named weekdays is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named weekdays


(lint/suspicious/noDuplicateObjectKeys)